exp/api/v1alpha4/zz_generated.conversion.go

```diff
@@ -1194,6 +1194,7 @@ func autoConvert_v1beta1_AzureManagedMachinePoolSpec_To_v1alpha4_AzureManagedMac
 	// WARNING: in.MaxPods requires manual conversion: does not exist in peer-type
 	// WARNING: in.OsDiskType requires manual conversion: does not exist in peer-type
 	// WARNING: in.EnableUltraSSD requires manual conversion: does not exist in peer-type
+	// WARNING: in.OSType requires manual conversion: does not exist in peer-type
```

Comment:
I think the tests are failing since we need to convert this manually. look for `OsDiskType` as an example

Reply:
fixed!